spikeinterface/core/old_api_utils.py

```
@@ -51,6 +51,176 @@ def create_extractor_from_new_recording(new_recording):
+
+        Returns
+        -------
+        spike_train: numpy.ndarray
+            An 1D array containing all the frames for each spike in the
+            specified unit given the range of start and end frames
+        """
+        return self._sorting.get_unit_spike_train(unit_id=self._unit_map[unit_id], segment_index=0,
+                                                  start_frame=start_frame, end_frame=end_frame)
+
+    def get_units_spike_train(self, unit_ids=None, start_frame=None, end_frame=None):
+        """This function extracts spike frames from the specified units.
+
+        Parameters
+        ----------
+        unit_ids: array_like
+            The unit ids from which to return spike trains. If None, all unit
+            spike trains will be returned
+        start_frame: int
+            The frame above which a spike frame is returned  (inclusive)
+        end_frame: int
+            The frame below which a spike frame is returned  (exclusive)
+
+        Returns
+        -------
+        spike_train: numpy.ndarray
+            An 2D array containing all the frames for each spike in the
+            specified units given the range of start and end frames
+        """
+        if unit_ids is None:
+            unit_ids = self.get_unit_ids()
+        spike_trains = [self.get_unit_spike_train(uid, start_frame, end_frame) for uid in unit_ids]
+        return spike_trains
+
+    def get_sampling_frequency(self):
+        """
+        It returns the sampling frequency.
+
+        Returns
+        -------
+        sampling_frequency: float
+            The sampling frequency
+        """
+        return self._sampling_frequency
+
+    def set_sampling_frequency(self, sampling_frequency):
+        """
+        It sets the sorting extractor sampling frequency.
+
+        Parameters
+        ----------
+        sampling_frequency: float
+            The sampling frequency
+        """
+        self._sampling_frequency = sampling_frequency
```

I would raise an error here.
We can't change the sampling rate on the fly in the new API.

---

or remove completely

---

Removed